stable/burrow/README.md

```diff
@@ -1,164 +1,183 @@
+| `readinessProbe.initialDelaySeconds` | start after | `5` |
+| `podAnnotations` | annotations to add to each pod | `{}` |
+| `podLabels` | labels to add to each pod | `{}` |
+| `affinity` | node/pod affinities | `{}` |
+| `tolerations` | list of node taints to tolerate | `[]` |
+| `nodeSelector` | node labels for pod assignment | `{}` |
 
 Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
 
-```console
-$ helm install stable/burrow --name my-release \
-  --set=image.tag=0.16.0,resources.limits.cpu=200m
+```bash
+helm install stable/burrow --name my-release \
+  --set=image.tag=0.23.1,resources.limits.cpu=200m
 ```
 
 Alternatively, a YAML file that specifies the values for the above parameters can be provided while installing the chart. For example,
 
-```console
-$ helm install stable/burrow --name my-release -f values.yaml
+```bash
+helm install stable/burrow --name my-release -f values.yaml
 ```
 
-> **Tip**: You can use the default [values.yaml](values.yaml)
-
 ## Runtime
 
-As noted above, there is a zero percent chance that you will want to deploy this chart with the default runtime configuration. When booting permissioned blockchains in a cloud environment there are three predominant considerations in addition to the normal configuration of any cloud application.
+It is unlikely that you will want to deploy this chart with the default runtime configuration. When booting permissioned blockchains in a cloud environment there are three predominant considerations in addition to the normal configuration of any cloud application.
 
 1. What access rights to place on the ports?
 2. What is the set of initial accounts and validators for the chain?
 3. What keys should the validating nodes have?
 
 Each of these considerations will be dealt with in more detail below.
 
-### Configuration of the Ports
+### Port Configuration
 
 Burrow utilizes three different ports by default:
 
-* `peer`: Burrow's peer port is the port on which the p2p communication within the blockchain network is conducted. The peer port is utilized by burrow's consensus engine (which is the [Tendermint](https://github.com/tendermint/tendermint) engine) to perform bilateral gossiping communication.
-* `rpc`: Burrow's rpc port is the port on which remote procedures are conducted. It is utilized by tools such as, e.g., the [Monax](https://github.com/monax/monax) tools which can be utilized to deploy smart contracts and perform other functions.
-* `api`: Burrow's api port is the port on which javascript libraries interact with the chain utilizing websockets. It is generally utilized by tools.
+* `Peer`: Burrow's peer port is used for P2P communication within the blockchain network as part of the consensus engine ([Tendermint](https://github.com/tendermint/tendermint)) to perform bilateral gossiping communication.
+* `RPC`: Burrow's RPC port is used for conducting remote procedures.
```

Comment:
for consistency let's `s/RPC/Info/g` -- this port is used for gaining information from a running burrow node over http calls with json returns.